eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -440,4 +440,7 @@
   <!-- The variables are assigned by jcommander annotation injection framework, cannot be made final. -->
   <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck"
             files="com.azure.perf.test.core.PerfStressOptions.java"/>
+            
+  <!-- This check doesn't apply / is a false positive. -->
+  <suppress checks="ModifierOrder" files="AuthenticationRecord.java"/>
```

Comment:
It's complaining about `private final static ObjectMapper OBJECT_MAPPER = new ObjectMapper();`. Changing `private final static` to `private static final` will fix the issue and you can remove this suppression.